app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ChargesController < ApplicationController
+
+  def create
+    # Amount in cents
+      @amount = 1500
```

Comment:
Excellent. This is a great technique to know about: saving dollar amounts as integers representing the cents.